lexnlp/ml/tests/test_model_card.py

```
@@ -0,0 +1,104 @@
+            clf,
+            self.tmpdir / "x",
+            metadata=md,
+        )
+        self.assertEqual(paths.model.stem, paths.card.stem)
+        self.assertEqual(paths.model.parent, paths.card.parent)
+
+
+class TestModelCardMetadataValidation(TestCase):
+    def test_required_description(self) -> None:
+        with self.assertRaises((TypeError, ValueError)):
+            ModelCardMetadata()  # type: ignore[call-arg]
```

Pylint (4.0.5)

[error] No value for argument 'description' in constructor call

(E1120)